guides/common/modules/con_top-performance-considerations.adoc

```diff
@@ -1,28 +0,0 @@
-[id="Top_Performance_Considerations_{context}"]
```

Comment:
You've deleted the hole `guides/common/modules/con_top-performance-considerations.adoc` file. If this was intentional, you'd also have to remove the line that includes this file which you can find using `git grep "con_top-performance-considerations.adoc"` and remove all the links to it's anchor.

Reply:
Hello, this file only in one place as of now and I have removed it from master.